### ISSUE TITLE
General colour scheme brightness/lightness/contrast review

### ISSUE THREAD

**Issue by ikirker** [UCL-RITS/mkdocs-rc-docs#56]
Even though some of the colours across the site don't fail accessibility review, they may not be sufficiently contrasting for good readability. This is an issue for taking a general look at the colour scheme and indicating areas we think could use improvement.

(It is not an issue for bike-shedding over e.g. whether you think purple is a good colour. (It is.))
